.github/workflows/_test.yaml

```diff
@@ -69,6 +69,8 @@ jobs:
                   {"target":"universal", "k8sVersion":"${{ env.K8S_MAX_VERSION }}"}
                 ],
                 "include":[
+                  {"embeddedDNS": "true", "k8sVersion": "${{ env.K8S_MAX_VERSION }}", "target": "kubernetes", "arch": "amd64"},
```

Comment:
Another option is to enable it on one Universal and Kubernetes zone of mulitzone test

Reply:
Yeah TBH as soon as I make sure this is stable I want to enable it by default and just have 1 fallback option